nodejs/README.rst

```diff
@@ -0,0 +1,34 @@
+===================
+Node.js Backend App
+===================
+
+A Node.js backend app using express and node-crate.
```

Comment:
node-crate?

Reply:
please provide a little bit more information. 
it would also be very informative which api endpoints are provided.

Reply:
as @turbo-ele pls provide more info here. At least link the used drivers/libs to its repos. Otherwise a user does not know wtf `node-crate` is.